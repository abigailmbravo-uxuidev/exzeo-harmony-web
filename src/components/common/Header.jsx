import React from 'react';
import logo from '../../img/TypTap.svg';

const Header = () => (
  <header>
    <div role="banner">
      <div id="logo" className="logo">
        <img src={logo} alt="TypTap Insurance" />
      </div>
      <button className="btn-icon btn-ellipsis-v"><i className="fa fa-ellipsis-v"></i></button>
      <nav>
        <a href="https://www.typtap.com">HOME</a>
        <a href="" className="active">AGENTS</a>
        <a href="https://www.typtap.com">POLICYHOLDERS</a>
        <a className="link-phone btn" href="tel:+844-289-7968"><i className="fa fa-phone" /><span>844-289-7968</span></a>
      </nav>
    </div>
  </header>
);

export default Header;
