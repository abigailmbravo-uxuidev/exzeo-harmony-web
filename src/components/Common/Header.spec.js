import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

it('renders without crashing', () => {
  const props = {
  };
  const wrapper = shallow(<Header {...props} />);
  expect(wrapper);
});
