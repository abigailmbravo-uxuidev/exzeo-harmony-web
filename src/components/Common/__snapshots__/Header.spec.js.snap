exports[`test renders without crashing 1`] = `
<header>
  <div
    role="banner">
    <button
      className="btn-icon btn-bars">
      <i
        className="fa fa-bars" />
    </button>
    <div
      className="logo"
      id="logo">
      <img
        alt="TypTap Insurance"
        src="TypTap.svg" />
    </div>
    <button
      className="btn-icon btn-ellipsis-v">
      <i
        className="fa fa-ellipsis-v" />
    </button>
    <nav
      className="fade-in">
      <a
        href="https://www.typtap.com">
        HOME
      </a>
      <a
        className="active"
        href="">
        AGENTS
      </a>
      <a
        href="https://policyholder.typtap.com/login">
        POLICYHOLDERS
      </a>
      <a
        className="link-phone btn"
        href="tel:+844-289-7968">
        <i
          className="fa fa-phone" />
        <span>
          844-289-7968
        </span>
      </a>
    </nav>
  </div>
</header>
`;
